python/src/adbc_example.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from dataclasses import dataclass
+from urllib.parse import parse_qs, urlparse
+
+from adbc_driver_flightsql import DatabaseOptions
+from adbc_driver_flightsql.dbapi import connect
+
+
+@dataclass
+class ConnAttr:
+    host: str
+    params: dict
+    auth_header: str
+
+
+def parse_jdbc_uri(uri):
+    parsed = urlparse(uri)
+    params = {k.lower(): v[0] for k, v in parse_qs(parsed.query).items()}
+    return ConnAttr(
+        host=parsed.path.replace("arrow-flight-sql", "grpc")
+        if params.pop("useencryption") == "false"
+        else parsed.path.replace("arrow-flight-sql", "grpc+tls"),
+        params=params,
+        auth_header=f"Bearer {params.pop('token')}",
+    )
+
+
+def main(args):
+    conn_attr = parse_jdbc_uri(args[1])
```

Comment:
I would refactor this out to read from an envvar - otherwise people are putting secrets in plain text into their command history, and using the script is a little clunky. I think having 

DBT_JDBC -> Full JDBC, optional
DBT_ENV_ID
DBT_ACCOUNT
DBT_PROJECT
DBT_AUTH_TOKEN

would allow us to easily save these things in a file and then source that to use the command. The command can then bed `dbt_sl 'select * from ...'`. 

We could also make this clever in other ways, and have a `--sql` flag for when we want raw SQL, or else have just a `--metric=revenue --dimensions=country` or whatever and have that build the SQL, same thing Hex would do. Don't have to do this, but it would be cool.

Reply:
The env var thing totally makes sense! I'll update both the python and java examples to use an ENV var for the JDBC string for now. For the rest of the flags and such, it'd definitely be cool to make some wrapper libraries in the future for different languages to make it easier to write integrations and/or clis!